deploy/update_plone

```diff
@@ -157,21 +160,40 @@ def run_buildout():
 
 
 def run_upgrades():
-    run_fg('bin/upgrade install --pick-site --proposed')
+    for site in get_sites():
+        run_fg('bin/upgrade install -s %s --proposed' % site['path'])
     return True
 
 
 def has_proposed_upgrades():
-    data = json.loads(run_bg('./bin/upgrade list --pick-site --upgrades --json'))
-    print 'Proposed upgrades:', len(data)
+    for site in get_sites():
+        command = './bin/upgrade list -s %s --upgrades --json' % site['path']
+        data = json.loads(run_bg(command))
+        if len(data) > 0:
+            print 'Proposed upgrades found.'
+            sys.stdout.flush()
+            return True
+
+    print 'Proposed upgrades: 0'
```

Comment:
Maybe change this to `No proposed upgrades found.`, as we never actually print the amount? What do you think?